modules/persistent-storage-local-pod.adoc

```diff
@@ -29,7 +29,7 @@ spec:
     - name: local-disks <1>
       mountPath: /data <2>
   volumes:
-  - name: localpvc
+  - name: local-disks
```

Comment:
Better to have indentations, others LGTM.

Reply:
Indentation should be consistent across the document, and I can see `volumeMounts` just above does not have spaces before `- name`.

I think it's better as it is now.